java/arcs/core/data/CapabilitiesNew.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+/** Store capabilities containing a grouping of individual capabilities. */
```

Comment:
I am not sure what this class is for. Can you give an example?

Is it essentially for a cartesian product of capabilites? e.g., TTL x Persistence, etc.

Reply:
rephrased a bit. it's not a cartesian product, because it may contain only a subset of all existing capabilities.